travis/travis_install_nightly

```diff
@@ -32,6 +32,11 @@ ln -s `which nodejs` $HOME/maintainer-quality-tools/travis/node
 npm install -g less less-plugin-clean-css
 
 
+# Update PhantomJS (v10 compat)
+npm install --prefix ${TRAVIS_BUILD_DIR} phantomjs-prebuilt@2.1.14
```

Comment:
We could remove the `@2.1.14` part and let it be the latest always, don't you think?
Well, I know this could lead to problems if API changes... Maybe `@2` would be enough (not sure how npm handles version strings...).